src/sgame/sg_main.cpp

```diff
@@ -2991,9 +2991,20 @@ void G_RunFrame( int levelTime )
 void G_PrepareEntityNetCode() {
```

Comment:
Note that `HandlePrepareNetcode` will never be allowed to do anything but exporting the component's state to the entity's network record. No member of the component may be changed. We just don't mark it `const` because it has an autogenerated declaration, as far as I know.

That being said, I think the only type of "wrong" the information can be is "outdated by one frame". And for clients, we're already talking about client frames (with asynchronous rate) that the server mimics, as far as I understand it. I do see that we may not want additional lag for some reason. I'm just hesitant to allow special cases in CBSE related code by default, because reducing the number of special cases to an elegant minimum is the point of CBSE. :stuck_out_tongue: